tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -71,6 +71,8 @@ public DashboardView()
 #if DEBUG
         Loaded += AddResetButton;
 #endif
+
+        Loaded += (s, e) => { this.Focus(FocusState.Programmatic); };

```

Comment:
Does this not happen for free since DashboardView is a ToolPage is a AutoFocusPage?

Reply:
Good catch.  Removed.